psa-crypto-sys/src/constants.rs

```diff
@@ -87,6 +90,8 @@ pub const PSA_KEY_USAGE_DECRYPT: psa_key_usage_t = 512;
 pub const PSA_KEY_USAGE_SIGN: psa_key_usage_t = 1024;
 pub const PSA_KEY_USAGE_VERIFY: psa_key_usage_t = 2048;
 pub const PSA_KEY_USAGE_DERIVE: psa_key_usage_t = 4096;
+pub const PSA_KEY_SLOT_COUNT: isize = 32;
+pub const PSA_MAX_PERSISTENT_KEY_IDENTIFIER: psa_key_id_t = 0x3fff_ffff;
```

Comment:
These values seem to be Mbed Crypto specific. Ideally this crate is common to all PSA Crypto implementation so it would be better, I think, to declare those values in the code that needs them directly.

Reply:
Yes, they are Mbed Crypto specific. Should they be moved to parsec?

Reply:
Yes I think it will be better.

Reply:
Presumably `PSA_MAX_PERSISTENT_KEY_IDENTIFIER` will be replaced with something else in PSA Crypto 1.0?

Reply:
> Presumably PSA_MAX_PERSISTENT_KEY_IDENTIFIER will be replaced with something else in PSA Crypto 1.0?

That's a good point, on section 9.1.1, for `psa_key_id_t`, it defines ranges of key identifiers. For example for application key identifiers, it can range from `PSA_KEY_ID_USER_MIN` (`0x00000001`) to `PSA_KEY_ID_USER_MAX` (`0x3fffffff`). Which is the same value as the one above 😄! So I guess as `PSA_KEY_SLOT_COUNT` should be removed, `PSA_MAX_PERSISTENT_KEY_IDENTIFIER` can be renamed to `PSA_KEY_ID_USER_MAX`